src/libsync/discovery.cpp

```diff
@@ -439,7 +439,7 @@ void ProcessDirectoryJob::processFileAnalyzeRemoteInfo(
     item->_modtime = serverEntry.modtime;
     item->_size = serverEntry.size;
 
-    auto postProcessServerNew = [item, this, path, serverEntry, localEntry, dbEntry] () mutable {
+    auto postProcessServerNew = [=, &path] () {
```

Comment:
This was a lambda because it was called twice before commit b9f17aeaf1d39c4a4fb1da3b15c1c9ea91b85bc8,  but i guess now the code no longer need to be in a lambda and can just be moved in the if bellow.